src/prodbg/viewdock/src/area/split.rs

```diff
@@ -119,18 +118,25 @@ impl Split {
             self.ratios[index + 1] - 0.05
         };
 
-        if res < min {
-            res = min;
+        if ratio < min {
+            ratio = min;
```

Comment:
That'll definitely help :). I'll seek if there is "clamp" function in std, if no - min/max is the best way.